main.js

```diff
@@ -0,0 +1,66 @@
+const button = document.getElementById("output");
+const fizzbuzzList = document.getElementById("fizzbuzzList");
+
+const listAdd = (a) => {
+  const li = document.createElement("li");
+  li.innerHTML = a;
+  fizzbuzzList.appendChild(li);
+  console.log(li);
+};
+
+button.addEventListener("click", () => {
+  const fizzNum = document.getElementById("fizzNum").value;
+  const buzzNum = document.getElementById("buzzNum").value;
+
+  if (fizzNum === "" || typeof fizzNum === "string") {
+    listAdd("整数値を入力してください");
```

Comment:
岩原さんも指摘をされていた通り
エラー発生時に後続の処理に進まないように `return` させる必要がありますよ。

```suggestion
    listAdd("整数値を入力してください");
    return;
```